haptools/data/genotypes.py

```diff
@@ -258,7 +258,7 @@ def _return_data(self, variant: Variant):
         data: npt.NDArray[np.uint8]
             Numpy array storing all genotypes
         """
-        return np.array(variant.genotypes, dtype=np.uint8)
+        return variant.genotype.array().astype(np.uint8)
```

Comment:
I'm using an undocumented feature of cyvcf2 here. I discovered it by reading the `tr_harmonizer` source code:

https://github.com/gymreklab/TRTools/blob/e0ed1ec62808b3a19596c0d60a30127cf6f0473f/trtools/utils/tr_harmonizer.py#L832

Apparently, you can have cyvcf2 return a numpy array directly, instead of converting from a list of lists. I figured that would be slightly faster?

Reply:
That does sound faster. That's a good find